api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -126,19 +125,20 @@
    * Converter function from backend representation (used with Hibernate) to client representation
    * (generated by Swagger).
    */
-  private static final Function<DbParticipantCohortStatus, ParticipantCohortStatus>
+  private static final BiFunction<
+          DbParticipantCohortStatus, Map<Long, String>, ParticipantCohortStatus>
```

Comment:
Do you mind just making this a vanilla function like `ParticipantCohortStatus toClientParticipant(participant, genderRaceEthMap)`? The hyper-functional stuff makes me think this is going to be more complicated than it needs to be. 

I can bring this up in Eng Sync.

Reply:
I'm not a huge fan of the Function/BiFunction and can change these to regular methods. I can do a follow up PR to convert all of these Function/BiFunctions instead of just changing the one in this PR. Assuming that everyone approves in Eng Sync.

Reply:
Thanks. 

To be clear, there's definitely a role for those classes as function parameters and map values, but we don't need to declare local ones in most cases (unless you need lambda-style capturing, which isn't super common, but I certainly don't mean to proscribe.